src/cart/cart.ts

```diff
@@ -1,9 +1,15 @@
 import { Cart } from "./types/cart.type";
+import { Product } from "../models/product.model";
+import { findProductQuery } from "../database/product.queries";
+import { logger } from "../logger/logger";
 
 const carts: Record<string, Cart> = {};
 
-export const addCart = (id: string) => {
-    carts[id] = {};
+export const registerNewCart = (id: string) => {
```

Comment:
revert - I'll explain why this happens in index.ts

Reply:
What do u mean by revert?